js/impress.js

```diff
@@ -397,6 +397,33 @@
                 scale:     1
             };
             
+            // Hide the mouse cursor in fullscreenmode after the configured
+            // amount of milliseconds without movement
+            if(rootData.hideMouseAfter) {
+                var mouseMoveTimer = null,
+                isCursorVisible = true;
+                
+                var hideCursor = function() {
+                    //isFullscreen
+                    if(!window.screenTop && !window.screenY){
+                        mouseMoveTimer = null;
+                        document.body.style.cursor = "none";
+                        isCursorVisible = false;
+                    }
+                };
+                
+                document.onmousemove = function() {
+                    if (mouseMoveTimer) {
+                        window.clearTimeout(mouseMoveTimer);
+                    }
+                    if (!isCursorVisible) {
+                        document.body.style.cursor = "default";
```

Comment:
If the user wanted to theme the cursor with css, you will now unset that. You need to save the original cursor value when hiding it, then set it back to original value here.